docs-source/content/userguide/managing-domains/model-in-image/overview.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Overview"
+date = 2020-03-11T16:45:16-05:00
+weight = 10
+pre = "<b> </b>"
+description = "Introduction to Model in Image, description of its runtime behavior, and references."
++++
+
+#### Content
+
+ - [Introduction](#introduction)
+ - [Runtime behavior overview](#runtime-behavior-overview)
+ - [Runtime updates overview](#runtime-updates-overview)
+ - [References](#references)
+
+#### Introduction
+
+Model in Image is an alternative to the operator's Domain in Image and Domain on PV domain types. See [Choose a Model]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}) for a comparison of operator domain types.
+
+Unlike Domain in PV and Domain in Image, Model in Image eliminates the need to pre-create your WebLogic domain home prior to deploying your domain resource.
+
+It enables:
+
+ - Defining a WebLogic domain home configuration using WebLogic Deploy Tool (WDT) model files and application archives.
+ - Embedding model files and archives in a custom Docker image, and using the WebLogic Image Tool (WIT) to generate this image.
+ - Supplying additional model files using a Kubernetes config map.
+ - Supplying Kubernetes secrets that resolve macro references within the models. For example, a secret can be used to supply a database credential.
+ - Updating WDT model files at runtime. For example, you can add a data source to a running domain. Note that all such updates currently cause the domain to 'roll' in order to take effect.
+ - Deploying standard WLS domains, Restricted JRF domains, or JRF domains.
+
+WDT models are a convenient and simple alternative to WebLogic WLST configuration scripts and templates. They compactly define a WebLogic domain using YAML files, plus support including application archives in a ZIP file. The WDT model format is described in the open source [WebLogic Deploy Tool](https://github.com/oracle/weblogic-deploy-tooling) GitHub project.
```

Comment:
Need a @rosemarymarano here on this sentence. :). The "plus support" phrase seems awkward.

Reply:
Yes, it's awkward. I had to read it twice before understanding the meaning. Better wording would be, "They compactly define a WebLogic domain using YAML files and support including application archives in a ZIP file."

Reply:
Ok. Will do.

Reply:
done.